src/Components/SmartComponents/Modals/CvePairStatusModal.js

```diff
@@ -113,10 +113,7 @@ export const CvePairStatusModal = ({ cves, updateRef, inventories, hasDifferentS
         title: intl.formatMessage(messages.cvePairStatusModalUpdateSuccessful)
     };
 
-    let modalTitle = intl.formatMessage(messages.cvePairStatusModalTitleSingle);
-    if (inventoryIds.length > 1 || cves.length > 1) {
-        modalTitle = intl.formatMessage(messages.cvePairStatusModalTitlePlural);
-    }
+    const modalTitle = intl.formatMessage(messages.cvePairStatusModalTitle, { count: inventoryIds.length * cves.length });
```

Comment:
Pair count = system count * cves count

Reply:
one(`inventoryIds`) or the other(`cves`) will be just one cause we don't have many-to-many edit.